Detectors/TrivialMC/include/DetectorsTrivialMC/O2TrivialMCApplication.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
```

Comment:
I would maybe not call the include dir "DetectorsTrivialMC". I think Detectors is misleading. What about "O2TrivalVMC" ?

Reply:
Should it still be in the `Detectors` directory in the source? Or would you propose to move it somewhere else?

Reply:
I think "Detectors" itself may be fine. There is already some other VMC stuff in there.

Reply:
I did some renaming. Does it make sense?